frontend/compositions/dashboardHeader/helperFunctions.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useCallback, useEffect } from "react"
+
+export function useMediaQuery(width: number): boolean {
+  const [targetReached, setTargetReached] = useState(false);
+
+  const updateTarget = useCallback((e) => {
+    if (e.matches) {
+      setTargetReached(true);
+    } else {
+      setTargetReached(false);
+    }
```

Comment:
We could simplify this as `useCallback((e) => { setTargetReached(e.matches) }, [])` if you like!
(or `!!e.matches` if "matches" isn't a boolean)